src/less/textbox/textbox.less

```diff
@@ -133,6 +125,14 @@ input.textbox__control--large {
     }
 }
 
+input.textbox__control:not(:read-only):focus,
+textarea.textbox__control:not(:read-only):focus {
+    .border-color-token(textbox-focus-border-color, color-stroke-default);
+    .background-color-token(textbox-focus-background-color, color-background-primary);
+    .color-token(textbox-focus-foreground-color, color-foreground-secondary);
```

Comment:
I believe this color is wrong.
In fact it should just not even be set, the color of the focused text should be the same as the unfocused.